source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -560,6 +572,9 @@ AdaptiveNamespaceStart
         [propget] HRESULT IsVisible([out, retval] boolean * value);
         [propput] HRESULT IsVisible([in] boolean value);
 
+        [propget] HRESULT FallbackType([out, retval] FallbackType * value);
+        [propget] HRESULT FallbackContent([out, retval] IAdaptiveCardElement ** content);
```

Comment:
Do we need putters for these as well?

Reply:
at the moment, at least, we're only setting content as an internal detail, so we peek over to the native type. that said, I suppose in order to use fallback via the object model (which is admittedly a *little* weird), you need to be able to set these.

---
In reply to: [263172755](https://github.com/Microsoft/AdaptiveCards/pull/2504#discussion_r263172755) [](ancestors = 263172755)